src/gridster-prototype/gridster-prototype.service.ts

```diff
@@ -183,25 +208,23 @@ export class GridsterPrototypeService {
     /**
      * Checks whether "element" position fits inside "containerEl" position.
      * It checks if "element" is totally covered by "containerEl" area.
-     * @param element Dragged element
-     * @param containerEl Element above which "element" is dragged
-     * @param event DraggableEvent
-     * @returns {boolean}
      */
-    private isOverGridster(item: GridsterItemPrototypeDirective, gridster: GridsterService, event): boolean {
+    private isOverGridster(item: GridsterItemPrototypeDirective, gridsterEl: HTMLElement, event, options): boolean {
         const el = item.$element;
-        const elContainer = gridster.gridsterComponent.$element;
-        const tolerance = gridster.options.tolerance;
+        const parentItem = <HTMLElement>gridsterEl.parentElement.closest('gridster-item');
```

Comment:
Sometimes, the parentElement on gridsterEl are null. I reported that on the issue #226